src/frontend/screens/Game/GamePage/index.css

```diff
@@ -5,8 +5,7 @@
   padding: var(--space-md);
   gap: var(--space-md);
   border-radius: 10px;
-  height: 100vh;
-  min-height: 100vh;
+  height: 100%;
```

Comment:
the controller hints div was overlapping the page content, hiding the `report a problem` button

![image](https://github.com/user-attachments/assets/e459ce1d-da31-4f61-9f78-ae408ff77dc8)

and now:
![image](https://github.com/user-attachments/assets/bcca054d-0c43-4e9f-9b31-615977a8881f)